src/components/Garden/ModalFlows/CreateProposalScreens/AddProposal.js

```diff
@@ -178,7 +185,7 @@ const AddProposalPanel = React.memo(({ setProposalData }) => {
     }
 
     if (link && !forumRegex.test(link)) {
-      errors.push('Forum post link not provided ')
+      errors.push('Forum post link not provided or not valid')
```

Comment:
Weird. It was removed. I'll check the PR #813

Reply:
Let's keep it 👍